cluster/gce/gci/configure-helper.sh

```diff
@@ -1840,6 +1840,16 @@ function update-node-label() {
   done
 }
 
+# A helper function that sets file permissions for kube-controller-manager to
+# run as non root.
+function run-kube-controller-manager-as-non-root {
+  prepare-log-file /var/log/kube-controller-manager.log ${KUBE_CONTROLLER_MANAGER_RUNASUSER:-2017} ${KUBE_CONTROLLER_MANAGER_RUNASGROUP:-2017}
+  setfacl -m u:${KUBE_CONTROLLER_MANAGER_RUNASUSER:-2017}:r "${CA_CERT_BUNDLE_PATH}"
```

Comment:
I suspect that `setfacl` is not present on COS images that run e2e tests here?

Reply:
Yes the master image for the tests was `cos-81-12871-59-0` the image that we want is `cos-81-12871-103-0`